hiera.yaml

```diff
@@ -2,11 +2,11 @@
 version: 5
 
 defaults:
-  datadir: "data"
+  datadir: 'data'
 
 hierarchy:
   - name: 'Yaml backend'
     data_hash: yaml_data
     paths:
-      - "nodes/%{trusted.certname}.yaml"
+      - 'nodes/%{trusted.certname}.yaml'
```

Comment:
does interpolation in hiera not require double quotes?

Reply:
The quoting is all just YAML. By the time Hiera gets it, it'll just be a Ruby data structure and string.